canonical-websites/jaas.ai
# Update links to not point to jujucharms.com

## Done

- Update URLs so they point to the local site, not jujucharms.com.
- External URLs are now stored as globals and referenced in the templates.
- Static files are referenced using the template tag.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Poke around the site, everything should link to your local site, not jujucharms.com (except for docs, discourse etc.).

## Details

- Fixes: #83.
